src/platform-includes/profiling/index/why-profiling/_default.mdx

```diff
@@ -0,0 +1,3 @@
+## Why profiling?
```

Comment:
```suggestion

```
I don't think this needs a separate heading. It should just be part of the introduction.